app/assets/stylesheets/image_x.scss

```diff
@@ -104,3 +104,32 @@ $image-thumb-slider-background-color: $sul-background;
   white-space: nowrap;
   width: 100%;
 }
+
+.#{$namespace}-image-x-side-control {
+  bottom: 0;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.